### PR TITLE
Add deno-yaml to database.json

### PR DESCRIPTION
**Pull request by bios21**
# Goal
This PR add a parsing library like native `JSON` to the registry.  
It's a port of [js-yaml](https://github.com/nodeca/js-yaml).

# Example
```ts
import * as YAML from 'https://deno.land/x/ParisDeno/deno-yaml/mod.ts';

const parsedObj = YAML.parse(str);
const dumpedStr = YAML.stringify(obj);
```

# CI
Yes (small, but yes .. :p)

### CODE REVIEW

File: src/database.json [hunk 1]
```diff
@@ -75,6 +75,10 @@
     "url": "https://raw.githubusercontent.com/lucascaro/denoversion/${b}/",
     "repo": "https://github.com/lucascaro/denoversion"
   },
+  "deno-yaml": {
```
Comment (j-f1) [inline src/database.json]: Why not just `"yaml"`?

Comment (bios21) [reply src/database.json]: To not be to much presumptuous. ^^'  
But if the key is just name rated and it's ok to use `yaml`, I can change it.

**Comment by bartlomieju** [comment]
:tada: nice! There is still this issue open in standard library - https://github.com/denoland/deno_std/issues/231 would you consider importing it there?

**Comment by bios21** [comment]
😲 Ok will do! Thank you for the proposal.

**Comment by ry** [comment]
@bios21 I agree it would be cool to have this in std

**Comment by bios21** [comment]
@ry Currently on it ;) https://github.com/denoland/deno_std/pull/528

(just fighting with eslint)

**Comment by bios21** [comment]
Closing in favor of https://github.com/denoland/deno_std/pull/528

**Pull request closed without merging**